shared/telemetry/custom_logging.py

```
@@ -0,0 +1,99 @@
+        _add_otel_context,
+        structlog.processors.StackInfoRenderer(),
+        structlog.processors.format_exc_info,
+    ]
+
+    if log_format == "json":
+        # 운영 환경 : Loki가 수집할 JSON 포맷
+        # 모든 필드가 JSON key-value로 출력
+        processors = shared_processors + [
+            structlog.processors.dict_tracebacks,
+            structlog.processors.JSONRenderer(),
+        ]
```

Ruff (0.15.12)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)